WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -551,6 +560,20 @@ class StoryComposerActivity : ComposeLoopFrameActivity(),
         return storyMediaFileDataList
     }
 
+    private fun buildStoryMediaFileDataForTemporarySlide(frame: StoryFrameItem, tempId: String): StoryMediaFileData {
+        val storyMediaFileData =
```

Comment:
This variable can be eliminated and the logic below inlined since it's only used in the return :)

Reply:
👍 done in 6bb0e31cf3